guides/user-guide.md

```diff
@@ -0,0 +1,67 @@
+
+- Understanding the risks facing your systems and services.
+- Developing services using best practices.
+- Checking that third party suppliers have sufficient and appropriate cyber security measures in place.
+- Applying IT Security policy to specific scenarios.
+
+## Asking for help
+
+If you need help dealing with a cyber security task or problem, send an email to:<br/>
+[cyberconsultancy@digital.justice.gov.uk](mailto:cyberconsultancy@digital.justice.gov.uk)
+
+Some requests are better handled by other teams. For example, to get help about a security incident, or for urgent matters, you should contact [security@digital.justice.gov.uk](mailto:security@digital.justice.gov.uk). Similarly, for help with data protection or GDPR questions, contact [dataprivacy@digital.justice.gov.uk](mailto:dataprivacy@digital.justice.gov.uk).
+
+The consultant team keep an eye open for email requests. Normally, you will get an acknowledgement or more detailed reply within two working days.
+
+To help us help you, please answer these questions in your email request, as best you can:
+
+1.  Who is the work for?
+2.  Why is it important (ie business case)?
+3.  What happens if the work is not done (or not done on time)?
+4.  What is the requirement (old-style accreditation on an existing contract, guidance or advice, review of proposed approach,...)?
+5.  What skills or experience do you believe is required to do the work (known skills/experience, or predicted skills/experience)?
+6.  When is the next milestone for the project, where cyber consulting input/involvement is requested?
+
+## How we handle requests for help
+
+Each working day, all new requests are reviewed.
+
+If the request is not appropriate for the team, a reply is sent promptly, explaining why it would be better to talk with a different team. We will usually recommend a more appropriate team in such cases, and provide contact details for them.
+
+If the request is appropriate for the consultants, some initial estimates are immediately carried out, such as:
+
+- Does the request affect many people or services across the MoJ? The wider the range, the greater the impact of the request.
+- Does the request require a small or large number of people, or time, to address? Some requests can be answered promptly by an email. Other requests might require a team of 2 or 3 people to be assigned for a 2 month project.
+- Does the request need certain skills or experience? This helps determine which consultant might be best placed to help.
+
+The answers to these questions help the team to decide whether a consultant can help immediately or not.
+
+If no consultant is available, the request is put on a 'next day refresh', meaning that it is considered again on the next working day. If a request stays in the 'next day refresh' state for too long - typically a week - it is automatically escalated to higher management for a review and decision.
```

Comment:
This para feels very 'internal' - could we not say that our our SLA is to reply to the requestor within one working day of receiving the request, although large / complex requests may require more information and discussion to work out the right way to support?

Reply:
Updated accordingly.